fastlane/lib/fastlane/helper/crashlytics_helper.rb

```diff
@@ -43,7 +43,11 @@ def generate_android_command(params)
 
           UI.user_error!("The `crashlytics_path` must be a jar file for Android") unless params[:crashlytics_path].end_with?(".jar") || Helper.test?
 
-          command = ["java"]
+          if ENV['JAVA_HOME'].nil?
+            command = ["java"]
+          else
+            command = ["#{ENV['JAVA_HOME']}/bin/java}"]
```

Comment:
The trailing "}" is breaking the plug in :(

Reply:
Ohh, nice catch, will do a follow-up PR, thanks

Reply:
https://github.com/fastlane/fastlane/pull/11257